azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -24,6 +25,19 @@ function createAzureClient(): AzureClient {
     return new AzureClient({ connection: connectionProps });
 }
 
+const createAzureClientWithScopes = (scopes: ScopeType[]): AzureClient => {
+    const connection: AzureLocalConnectionConfig = {
+        tokenProvider: new InsecureTokenProvider("fooBar", generateUser(), scopes),
```

Comment:
How are you consuming InsecureTokenProvider changes here? Did you publishe them?

Reply:
I did not publish them, but I modified the node_modules and tests passed! Files modified; 
`
azure/node_modules/@fluidframework/test-runtime-utils/src/insecureTokenProvider.ts
azure/node_modules/@fluidframework/test-runtime-utils/dist/insecureTokenProvider.js
azure/node_modules/@fluidframework/test-runtime-utils/dist/insecureTokenProvider.d.ts
`

Reply:
It looks like you imported relevant changes needed to run your tests without hacking node modules. Please resolve this if so.